bin/ch/WScriptJsrt.cpp

```diff
@@ -325,11 +334,46 @@ JsErrorCode WScriptJsrt::InitializeModuleInfo(JsValueRef specifier, JsModuleReco
     return JsNoError;
 }
 
-JsErrorCode WScriptJsrt::LoadModuleFromString(LPCSTR fileName, LPCSTR fileContent)
+bool WScriptJsrt::GetFullPath(LPCSTR fileName, __out char* const fullPathNarrow)
+{
+    size_t len = 0;
+    if (fileName)
+    {
+        if (_fullpath(fullPathNarrow, fileName, _MAX_PATH) == nullptr)
+        {
+            return false;
```

Comment:
Looks like we are in WScriptJsrt::LoadScript, but not in FetchImportedModuleHelper

---
In reply to: [125370693](https://github.com/Microsoft/ChakraCore/pull/3284#discussion_r125370693) [](ancestors = 125370693)